client/webserver/site/src/html/order.tmpl

```diff
@@ -212,7 +212,7 @@
       <div class="text-start">[[[acceleration_transactions]]]</div>
       <div class="fs14 text-start">
         {{range $ord.AccelerationCoins}}
-          <a target="_blank" class="plainlink mono" data-explorer-id="{{$ord.FromID}}" data-explorer-coin="{{.StringID}}">{{.StringID}}</a><br>
+          <a target="_blank" class="plainlink mono" data-asset-id="{{$ord.FromID}}" data-explorer-coin="{{.StringID}}">{{.StringID}}</a><br>
```

Comment:
Renaming to better reflect the assumptions we make (**explorer-id**=**asset-id**).

Reply:
What if we need to tag other elements with an asset ID. **This** asset ID is specifically for block explorer links. The attribute name reflected that.

Reply:
Tend to agree.  We've named this because if its application, not the assumption of the data type.  Let's just leave it.

Reply:
Renamed so it's like it was previously: https://github.com/decred/dcrdex/pull/2181/commits/2b6ba04c7aa2f5e07c741f8c5a5be16fff208340